docs/content/manual/_incoming/1707-zero-downtime-upgrade.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Zero downtime upgrade 
+---
+
+* Related issues: [#1707](https://github.com/harvester/harvester/issues/1707) [BUG] Zero downtime upgrade stuck in "Waiting for VM live-migration or shutdown..."
+
+## Category: 
+* Upgrade
+
+## Verification Steps
+1. Create a ubuntu image from URL
+1. Enable Network with management-mgmt
+1. Create a virtual network vlan1 with id 1
+1. Setup backup target
+1. Create a VM backup
+1. Follow the [guide](https://github.com/harvester/docs/blob/42598035cfe79856f7e159d4c47c4d7d29178900/docs/upgrade/automatic.md) to do upgrade test
```

Comment:
Should we update the link to this one?
https://github.com/harvester/docs/blob/main/docs/upgrade/automatic.md

Reply:
Done